game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -33,13 +39,22 @@ class Cell(id: Int, startPosition: Vector2 = new Vector2(0f, 0f)) {
   def velocity_=(vel:Vector2) {
     _velocity = if (vel.magnitude < maxSpeed) vel else vel.normalize * maxSpeed
   }
+  def mass = _mass
+  def mass_=(m: Float) {
+    _mass = max(m, Cell.MinMass)
+  }
 
 
   def update(deltaSeconds: Float) {
+    mass -= massDecay(deltaSeconds)
     velocity += acceleration * deltaSeconds
     position += velocity * deltaSeconds
   }
 
+  def massDecay(deltaSeconds: Float) = {
+    mass * pow(Cell.MassDecayPerSecond, deltaSeconds).toFloat
```

Comment:
Not sure about this `pow(Cell.MassDecayPerSecond, deltaSeconds)`

The way the formula is working is as the smaller the `deltaSeconds` is, the more mass will be removed from the cell. Not sure it is the right behavior! 

Just try with a value above 1 second, it will almost remove nothing.